src/ts/uimanager.ts

```diff
@@ -127,7 +128,8 @@ export class UIManager {
     }
 
     this.player = player;
-    this.config = {
+    this.uiconfig = config;
+    this.mergedConfig = {
```

Comment:
Why were these changes necessary? It seems to me as if the only thing necessary was to fix the markers update in `updateConfig`?

Reply:
To clarify what the config is actually and to facilitate the handling when the config is updated.

Reply:
Still the `uiconfig` field is never used except in the constructor which has access to the `config` parameter anyway, hence the field is unnecessary.